content/docs/pulumi-cloud/admin/self-hosted/components/api.md

```diff
@@ -454,3 +454,9 @@ In order to enable [Pulumi Deployments](/docs/pulumi-cloud/deployments), the fol
     version: v0.0.5
     service_url: https://pulumiapi.acmecorp.com
     ```
+
+## Pulumi ESC
+
+In order to enable [Pulumi ESC](/docs/esc), the following must be configured:
```

Comment:
this might be implied based on the encryption section above but we might want to call out that you will need one of the encryption env variables set as well